Update New-Team.md (#8220)
MicrosoftDocs/office-docs-powershell: teams/teams-ps/teams/New-Team.md

```

## DESCRIPTION
Creates a new team with user specified settings, and returns a Group object with a GroupID property. 

> > [!WARNING]
> >From version 2.5.0, the return type of this cmdlet has been modified to a resource location for the operation. This resource can be polled for getting the status of the operation using the Get-Operation cmdlet.

Reviewer: ```suggestion
> [!NOTE]
> From Teams version 2.5.0, the return type of this cmdlet has been modified to a resource location for the operation. This resource can be polled to get the status of the operation using the Get-Operation cmdlet.
```

Author: @JohanFreelancer9 thanks a lot for the review. Actually, this is the powershell module version, so could we keep it as just version instead of Teams version? Also, this will be a breaking change for admins so we want to show it as a warning than just a note. Please let me know if this would be okay.

Reviewer: @shriya999, thanks for your response. Your explanation makes sense, so I have no objection to keeping it as is. I'll delete my suggestion. Thanks again.

> >From version 2.5.0, the return type of this cmdlet has been modified to a resource location for the operation. This resource can be polled to get the status of the operation using the **Get-Operation** cmdlet. For reference, see Example 4 .

Reviewer 2: ```suggestion
>From version 2.5.0, the return type of this cmdlet has been modified to a resource location for the operation. This resource can be polled to get the status of the operation using the Get-Operation cmdlet. For reference, see Example 4 .
```

This example creates a team, adds three members to it, and creates three channels within it.

### Example 4
```
$output = New-Team -Displayname 'TestTeam'
$groupId = $output.TargetResourceLocation.Split("'")[1]
$operationId = $output.TargetResourceLocation.Split("'")[3]
Get-Operation -GroupId $groupId -OperationId $operationId
```

> Note: This is applicable for versions 2.5.0 and above. This example creates a team and gets the status of the team creation operation.

Reviewer 2: ```suggestion
> [!NOTE] 
> This is applicable for versions 2.5.0 and later. This example creates a team and gets the status of the team creation operation.
```
